cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureDefBuilder.java

```diff
@@ -231,27 +221,14 @@ private MeasureScoring getMeasureScoring(Measure measure) {
         return null;
     }
 
-    private MeasureScoring getGroupMeasureScoring(MeasureScoring measureLevelScoring, MeasureGroupComponent group) {
-        // see if group component has scoring Url
-        var scoringExtension = group.getExtensionByUrl(CQFM_SCORING_EXT_URL);
-        if (scoringExtension != null) {
-            // extract scoringType
-            CodeableConcept coding = (CodeableConcept) scoringExtension.getValue();
-            return MeasureScoring.fromCode(coding.getCodingFirstRep().getCode());
-        }
-        // otherwise return measureLevelScoring
-        return measureLevelScoring;
-    }
-
-    private boolean isIncreaseImprovementNotation(CodeableConcept improvementNotationValue) {
-        validateImprovementNotationCode(improvementNotationValue);
-        return improvementNotationValue.hasCoding(
-                MEASUREREPORT_IMPROVEMENT_NOTATION_SYSTEM, IMPROVEMENT_NOTATION_SYSTEM_INCREASE);
+    private MeasureScoring getMeasureScoring(Measure measure) {
```

Comment:
General suggestion:  Mark method parameters you expect to be null with @Nullable.